### PR TITLE
Fix switch auto discovery

### PR DESCRIPTION
**Pull request by dalebremner**

### CODE REVIEW

File: test/tests/api/v2_0/nodes_tests.py [hunk 1]
```diff
@@ -31,31 +31,31 @@ def __init__(self):
         self.__test_nodes = [
             {
                 'identifiers': ["FF:FF:FF:01"],
-                'autoDiscover': 'false',
+                'autoDiscover': False,
```
Comment (hohene) [inline test/tests/api/v2_0/nodes_tests.py]: Though it looks correct, I get a 404 error when using this in the payload, says create node is expecting a 'string' not a 'boolean'.  

ApiException: (400)
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'Content-Length': '149', 'X-Powered-By': 'Express', 'Connection': 'keep-alive', 'ETag': 'W/"95-yvHMS5tuMeGFozsXrC0ZAgWiHzk"', 'Date': 'Wed, 03 May 2017 11:44:29 GMT', 'Access-Control-Allow-Origin': '*', 'Content-Type': 'application/json; charset=utf-8'})
HTTP response body: {"message":"Validation errors","status":"400","UUID":"99f0efcd-32c9-49b4-ac92-98d10cc02c54","errors":["Expected type string but found type boolean"]}

**Comment by dalebremner** [comment]
Jenkins: depends on https://github.com/RackHD/on-http/pull/637

**Comment by dalebremner** [comment]
Jenkins: test this please

**Comment by dalebremner** [comment]
https://rackhd.atlassian.net/browse/RAC-4731

**Pull request merged**